datasync/chaindatafetcher/kas/repository.go

```diff
@@ -0,0 +1,55 @@
+package kas
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jinzhu/gorm"
+	"github.com/klaytn/klaytn/log"
+	"time"
+)
+
+const (
+	maxTransactionCount         = int64(1000000)
+	maxTransactionLogCount      = int64(100000)
+	maxInternalTransactionCount = int64(10000)
```

Comment:
Does `maxInternalTransactionCount` mean literally the maximum internal transaction count? If it does, why this value is smaller than `maxTransactionCount`? I think `maxTransactionCount` should be the smallest value among them.

Reply:
These values are predefined in KAS for unique id.